media/css/firefox/new/fx-lifestyle/conformity-not-default.less

```diff
@@ -14,6 +15,10 @@
         p {
             .font-size(22px);
         }
+
+        .copy {
+            width: 500px;
```

Comment:
We could totally change these to use `background-size: cover` which would be more flexible, but given that these are in english only I decided against it for now.